src/app/public/modules/overlay/overlay-host.component.scss

```diff
@@ -0,0 +1,15 @@
+:host {
+  position: fixed;
+  z-index: 1000;
```

Comment:
Should we put a comment inline here to define why `1000` is significant?

Reply:
Yep, once I figure out why it's significant! :-P I'll work on that...